norman_objects/shared/models/model_preview.py

```diff
@@ -13,3 +15,6 @@ class ModelPreview(NormanBaseModel):
     creation_time: datetime = Field(default_factory=lambda: datetime.now(timezone(timedelta(0))))
     short_description: str
     assets: list[ModelAsset] = []
+
+    model_tags: list[ModelTag] = []
+    user_added_tags: list[ModelTag] = []
```

Comment:
Design: do we want use tags for each preview? Probably only relevant when opening a specific model

Reply:
@avremy-nai 
Had a typo. Should have been `user_added_tags`